src/lib.rs

```diff
@@ -224,7 +279,27 @@ pub trait Resampler<T>: Send {
 pub trait VecResampler<T>: Send {
     /// Resample a chunk of audio.
     /// Input and output data is stored in vectors, where each element contains a vector with all samples for a single channel.
-    fn process(&mut self, wave_in: &[Vec<T>]) -> ResampleResult<Vec<Vec<T>>>;
+    fn process(
+        &mut self,
+        wave_in: &[Vec<T>],
+        active_channels_mask: Option<&[bool]>,
+    ) -> ResampleResult<Vec<Vec<T>>>;
+
+    /// Resample a chunk of audio, into a pre-allocated output buffer.
+    /// Input and output data is stored in vectors, where each element contains a vector with all samples for a single channel.
```

Comment:
```suggestion
    /// Input and output data is stored in vectors, where each element contains a vector with all samples for a single channel (deinterleaved).
```

Reply:
Ideally I'd like libraries to be generic with respect to interleaved vs deinterleaved, but that's [another topic](https://github.com/udoprog/audio/issues/3)...